crates/typos-cli/src/file.rs

```diff
@@ -683,6 +764,42 @@ fn fix_file_name<'a>(
     Ok(new_path)
 }
 
+fn select_fix(typo: &typos::Typo<'_>) -> Option<usize> {
+    let corrections = match &typo.corrections {
+        typos::Status::Corrections(c) => c,
+        _ => return None,
+    }
+    .clone();
+
+    if corrections.len() == 1 {
+        let confirmation = Confirm::new()
+            .with_prompt("Do you want to apply the fix suggested above?")
+            .default(true)
+            .show_default(true)
+            .interact()
+            .unwrap();
+
+        if confirmation {
+            return Some(0);
+        }
+    } else {
+        let mut items = corrections.clone();
```

Comment:
nit: if we've cloned the corrections on 772, why do we need this clone?  We could just do `let mut items = corrections;`